packages/blade/package.json

```diff
@@ -1,5 +1,13 @@
 {
   "name": "@razorpay/blade",
   "version": "0.0.1",
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
+    "@types/react": "17.0.3",
+    "@types/react-native": "0.64.2",
+    "blade": "3.3.1",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-native": "0.64.0"
```

Comment:
Root `package.json` has react native 0.61.5. This is causing our builds to fail. 

Degrade this, upgrade root or let's hack it and make it work with each other.